Makefile

```diff
@@ -61,7 +61,7 @@ GIT_TAG ?= $(shell git describe --tags --dirty --always)
 IMAGE_TAG ?= $(IMAGE_REPO):$(GIT_TAG)
 
 # WEB_ROOT defines path that will open web UI.
-WEB_ROOT ?= /web/
+WEB_ROOT ?= /
```

Comment:
Do we want to remove the web root? We can do it later if you think we can remove it since everything is served from `/` now

Reply:
Yep, it is cleaner without it. I have removed it.